sdk/storage/Azure.Storage.Queues/api/Azure.Storage.Queues.netstandard2.0.cs

```diff
@@ -1,5 +1,12 @@
 namespace Azure.Storage.Queues
 {
+    public partial class InvalidMessageEventArgs : Azure.SyncAsyncEventArgs
+    {
+        public InvalidMessageEventArgs(Azure.Storage.Queues.QueueClient queueClient, Azure.Storage.Queues.Models.PeekedMessage message, bool runSynchronously, System.Threading.CancellationToken cancellationToken) : base (default(bool), default(System.Threading.CancellationToken)) { }
+        public InvalidMessageEventArgs(Azure.Storage.Queues.QueueClient queueClient, Azure.Storage.Queues.Models.QueueMessage message, bool runSynchronously, System.Threading.CancellationToken cancellationToken) : base (default(bool), default(System.Threading.CancellationToken)) { }
+        public object Message { get { throw null; } }
+        public Azure.Storage.Queues.QueueClient QueueClient { get { throw null; } }
```

Comment:
What do folks think about  calling this `Queue` instead of `QueueClient`?  I was leaning toward `Queue` because I think everything's still really clear when we have `{ Queue, ReceivedMessage, PeekedMessage }`.  I don't feel strongly though.

Reply:
I like `Queue`